### ISSUE TITLE
boost: incorrect install directory set

### ISSUE THREAD

**Issue by brucellino** [SouthAfricaDigitalScience/boost-deploy#3]
From build 20, we have the error : 

```
ls: cannot access /apprepo/generic/u1404/x86_64/boost/1.59.0-mpi-1.8.8-gcc-4.9.2: No such file or directory
```

This is from a line in `check-build.sh` which tries to see what's in `${BOOST_DIR}`, which is apparently incorrectly set.

**Comment by brucellino** [comment]
Changed the modulefile to use the SOFT_DIR environment variable.

**Comment by brucellino** [comment]
fixed in e75cad3a1

**Comment by brucellino** [comment]
included in build 46
